src/Migrations/AddXHPChildrenDeclarationMethodMigration.hack

```diff
@@ -43,19 +43,14 @@ final class AddXHPChildrenDeclarationMethodMigration
     $before = C\firstx($decls->getChildrenByType<IDeclaration>());
     $t = $before->getFirstTokenx();
     $use = self::getUseNamespace();
-    if ($t->getLeading() !== $t->getLeadingWhitespace()) {
-      $ut = $use->getFirstTokenx();
-      return $in->withDeclarations(
-        $in->getDeclarations()->insertBefore(
-          $before,
-          $use->replace($ut, $ut->withLeading($t->getLeading())),
-        ),
-      )
-        ->replace($t, $t->withLeading(null));
-    }
```

Comment:
Doesn't look like these changes are meant to be in this PR?

Reply:
This change is to fix a 5607 linter error. I reviewed `$t->getLeading() !== $t->getLeadingWhitespace()` and I believe the condition is always true, therefore, I think the `if` clause is unnecessary.

Reply:
Anyway, I will split the change to another PR